Feature/scan 137/updating docs (#123)
stackhawk/hawkscan-action: QUICKSTART.md

> ## StackHawk Setup

Reviewer: I believe all of this information should be in HawkDocs instead, for a couple of reasons:
1. we already have hawkscan quickstart docs in hawkdocs - https://docs.stackhawk.com/hawkscan/
2. we already have hawkscan-action docs in hawkdocs - https://docs.stackhawk.com/continuous-integration/github-actions.html
3. Having the same points covered in many places will be harder to maintain than just linking to the canonical info
4. Most users of the hawkscan action won't see this repo - they will read our action's docs from the Marketplace site, which only shows the top-level README.md - https://github.com/marketplace/actions/stackhawk-hawkscan-action

I think we should link out to the hawkscan quickstart, and the hawkscan-action quick start, from the parent README.md in this repo, which gets published as the documentation in the GitHub Marketplace.

> 1. If you do not have an existing account or application on StackHawk.
>    [Follow the steps getting set up](https://docs.stackhawk.com/hawkscan/#quickstart). Make sure to save your API key

Reviewer 2: these sentences read a little odd to me.

Maybe something like this?
```suggestion
1. If you do not have an existing account or application on StackHawk,
   [follow our platform quickstart guide](https://docs.stackhawk.com/hawkscan/#quickstart). Make sure to save your API key
```

2. [Save the API Key](https://app.stackhawk.com/settings/apikeys)

## GitHub Setup

1. Create or use an existing GitHub [Repository](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository).

2. Add the following secrets to your repository's [secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets)

>     - `API_KEY`: API Key from your stack hawk platform

Reviewer 2: I think we just did a change to call this `HAWK_API_KEY` to be consistent in all the places

```suggestion
    - `HAWK_API_KEY`: API Key from your stack hawk platform
```
